doc/api/net.md

```diff
@@ -296,7 +296,7 @@ added: v0.1.90
 
 Start a [IPC][] server listening for connections on the given `path`.
 
-#### server.listen([port][, host][, backlog][, callback])
+#### server.listen([[[port[, host[, backlog]]][, callback])
```

Comment:
If I count correctly, for the `port, host, backlog` we have 5 `[` and 3 `]`, is it OK?

Reply:
Definitely not correct. Thanks for spotting.

Reply:
Maybe it is just:
```
server.listen([port[, host[, backlog]]][, callback])
```
here and above?